modules/t/test-genome-DBs/homology/compara/table.sql

```diff
@@ -388,8 +388,14 @@ CREATE TABLE `homology` (
   KEY `gene_tree_root_id` (`gene_tree_root_id`)
 ) ENGINE=MyISAM AUTO_INCREMENT=100990070 DEFAULT CHARSET=latin1;
 
+CREATE TABLE `homology_id_mapping` (
+  `curr_release_homology_id` int(10) unsigned NOT NULL,
+  `prev_release_homology_id` int(10) unsigned NOT NULL,
+  `mlss_id` int(10) unsigned NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

Comment:
That's because `homology_id_mapping` comes from `pipeline-tables.sql` which uses InnoDB for all tables (as opposite to `table.sql` which uses MyISAM by default).
BTW this table can be removed from the schema because it's not used any more, see https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-2788

Reply:
Oh, wait, actually @carlacummins you've already removed it :smile: ! https://github.com/Ensembl/ensembl-compara/commit/d12ebc9ce153891e86c88be3f75e30840e686971 . 
@CristiGuijarro can you remove references to this table from all the test databases ? (and maybe assign the Jira ticket to yourself and explain what's actually needed)

Reply:
Oh, I just misread that and created the ticket linked. I thought you meant Carla had already removed it... Can someone delete the Jira ticket I just made?